pxtlib/semver.ts

```diff
@@ -101,6 +101,10 @@ namespace pxt.semver {
         return aa.major - bb.major;
     }
 
+    /**
+     * Compares two semver version strings and returns -1 if a < b, 1 if a > b and 0
+     * if versions are equivalent
+     */
```

Comment:
maybe worth a note on what happens if a / b are nullish since that seemed to cause an issue previously? but already better than not having a comment ~

Reply:
updated the comment, take a look